src/shared/styles/variables/palette.ts

```diff
@@ -33,6 +33,7 @@ export const palette = {
   on_surface_alfa38: 'rgba(26, 28, 30, 0.38)',
   on_surface_variant_alfa8: 'rgba(66, 71, 78, 0.08)',
   on_surface_variant_alfa12: 'rgba(66, 71, 78, 0.12)',
+  on_surface_variant_alfa16: 'rgba(66, 71, 78, 0.16)',
```

Comment:
@farmerpaul is there a process to how we go about adding colours? This colour is to address the disabled state on the checkbox but I wasn't sure if we have some method behind the pallet colours. 🤔

Reply:
That's what I'd do. There's no system in place of syncing tokens from Figma to the code. Just whenever you see a missing one, add it just like you've done here! :+1: